www/templates/forms.html

```diff
@@ -115,6 +115,11 @@
         <span title="Checking this box will force the build to be rebuilt, regardless of whether or not there is a previous successful build at the same revision. If the build triggers dependencies, those will also be rebuilt." class="tooltip question-icon">
       </span>
        <input type='checkbox' name='checkbox' value='{{f.fullName}}' {{default_props[sch.name+"."+f.fullName]}} />
+    {% elif 'Smart Select' in f.label%}
```

Comment:
@alex-gubernsky it looks like at some point we broke the support for generic bool types in the force build form, could we do something like this instead?, later we will add the support to add the tooltip based on what you submit in configuration

    {% elif 'bool' in f.type %}
       <span class="label">{{f.label}}</span>